drivers/counter/Kconfig.nrfx

```diff
@@ -130,6 +140,16 @@ config COUNTER_RTC1_PRESCALER
 	help
 	  Frequency = 32768 / (prescaler+1).
 
+config COUNTER_RTC1_PPI_WRAP
+	bool "Enable PPI wrapping in RTC1"
+	select NRFX_PPI
+	select NRFX_DPPI
```

Comment:
Don't we need here `if HAS_HW_NRF_PPI` and `if HAS_HW_NRF_DPPIC`, respectively, to avoid Kconfig warnings?

Reply:
done, though there where no warnings.